src/main/java/org/graylog2/syslog4j/impl/unix/socket/UnixSocketSyslog.java

```diff
@@ -39,6 +40,12 @@ public void setSunPath(String sunPath) {
             System.arraycopy(sunPath.getBytes(), 0, this.sun_path, 0, sunPath.length());
             System.arraycopy(ZERO_BYTE, 0, this.sun_path, sunPath.length(), 1);
         }
+
+		@Override
+		protected List getFieldOrder() {

```

Comment:
The Javadoc at https://jna.java.net/nonav/javadoc/com/sun/jna/Structure.html#getFieldOrder() doesn't mention this method being `abstract` and "my" JNA implementation provides a standard implementation for this method.

The question is, if is has to be overridden (it hasn't been overridden before this PR), and if it has to, whether it should return `null` (which might cause NPEs across the board) or simply return an empty list.